src/components/Button/__tests__/button-tests.jest.js

```diff
@@ -1,14 +1,15 @@
 import React from "react";
-import { render, fireEvent, cleanup, act } from "@testing-library/react";
-import { sinon, expect } from "../../test/test-helpers";
-import Button from "./Button";
+import { render, fireEvent, cleanup } from "@testing-library/react";
+import { sinon, expect } from "../../../test/test-helpers";
```

Comment:
please revert the changes for the button component tests (we do not want to do any changes in this component tests right now)